plugins/TagFix_Postcode.py

```diff
@@ -81,7 +81,7 @@ def init(self, logger):
         self.errors[31901] = self.def_class(item = 3190, level = 3, tags = ['postcode', 'fix:chair'],
             title = T_('Invalid postcode'),
             detail = T_(
-'''Check postcode as decribed on
+'''Check postcode as described on
```

Comment:
Suggestion: check *the* postcode as described on

Reply:
Resolved by https://github.com/osm-fr/osmose-backend/pull/1704/commits/60730269ed49c7f4a75d0058b0ccd6fae25b79e1, alt. "Format the postcode ..."